### ISSUE TITLE
Allow editor to directly open *.castle-user-interface

### ISSUE THREAD

**Issue by peardox** [castle-engine/castle-engine#318]
Not sure if this is meant to work and is now broken or was never implemented...

At present opening a .castle-user-interface file does this...
![castle-user-interface](https://user-images.githubusercontent.com/1549219/125171703-6715b600-e1ad-11eb-950d-5dec55b191d3.png)

It would be rather cool if it loaded the owning CastleEngineManifest.xml that is (usually) in the directory above then loaded the requested *.castle-user-interface

If this is a new feature then it's a desirable one and should be fairly simple to implemement.

OK - you could just open the owning CastleEngineManifest.xml but then manually open the design - not the point, user friendliness is (and the error message above is not exactly user friendly)

Additionally it would be a nice addition of there was a Recently Used designs list and button to open as is the case with the MRU for the CastleEngineManifest.xml files

**Comment by michaliskambi** [comment]
Indeed, opening on the command-line of the editor `xxx.castle-user-interface` (or similar `xxx.castle-transform` or `xxx.castle-component`) can be added. It's a low priority TBH -- in a normal use-case, you work within already-opened instance of the editor for a long time. Double-click in editor on `xxx.castle-user-interface` etc. to open it.

> Recently Used designs

Yes, this makes sense too.
